field_bundle/FieldBundleGet.F90

```diff
@@ -53,6 +54,7 @@ subroutine bundle_get(fieldBundle, unusable, &
       character(:), optional, allocatable, intent(out) :: standard_name
       character(:), optional, allocatable, intent(out) :: long_name
       type(StateItemAllocation), optional, intent(out) :: allocation_status
+      logical, optional, intent(out) :: do_regrid_transform
```

Comment:
I'm thinking that we can probably find a better name here, as it does not quite reflect the intent.

We are not so much telling upstream to regrid (e.g., if the geoms are the same, it should not trigger a regrid), as saying that the bundle data has been updated.

Something like `bracket_data_changed`.  (Not wed to the details here.)